src/input_common/drivers/tas_input.cpp

```diff
@@ -105,10 +105,16 @@ void Tas::LoadTasFile(size_t player_index, size_t file_index) {
             continue;
         }
 
-        const auto num_frames = std::stoi(seg_list[0]);
-        while (frame_no < num_frames) {
-            commands[player_index].emplace_back();
-            frame_no++;
+        try {
+            const auto num_frames = std::stoi(seg_list[0]);
+            while (frame_no < num_frames) {
+                commands[player_index].emplace_back();
+                frame_no++;
+            }
+        } catch (std::invalid_argument) {
```

Comment:
```suggestion
        } catch (const std::invalid_argument&) {
```

Otherwise this copies the exception